stl/src/uncaught_exception.cpp

```diff
@@ -4,9 +4,9 @@
 #include <eh.h>
 #include <exception>
 _STD_BEGIN
-
+#if _HAS_DEPRECATED_UNCAUGHT_EXCEPTION

```

Comment:
For binary compatibility, `src` must always export `uncaught_exception()` from the DLL, even if user headers decide not to declare it.

Since this definition says `_CRTIMP2_PURE` (which will expand to `__declspec(dllexport)`), I believe that simply not adding guards here is sufficient. If we needed the declaration for some reason, we would have to alter the STL's build system to compile with the fine-grained escape hatch (if not the coarse-grained), but that shouldn't be necessary.

Reply:
Removed the guards